scripts/estimate_memory_usage.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python3
+
+import argparse
+import math
+
+base_mem_requirement = 32
+
+parser = argparse.ArgumentParser(description='Estimate memory needed to run H2O for a given dataset.')
+parser.add_argument('--nrows', dest='nrows', required=True, type=int,
+                   help='count of rows in the dataset (required)')
+parser.add_argument('--ncols', dest='ncols', required=True, type=int,
+                   help='count of columns of all types in the dataset (required)')
+parser.add_argument('--n-num-cols', dest='n_num_cols', required=False, type=int, default=0,
+                   help='count of numeric collumns in the dataset')
+parser.add_argument('--n-string-cols', dest='n_string_cols', required=False, type=int, default=0,
+                   help='count of string collumns in the dataset')
+parser.add_argument('--n-uuid-cols', dest='n_uuid_cols', required=False, type=int, default=0,
+                   help='count of UUID collumns in the dataset')
+parser.add_argument('--n-cat-cols', dest='n_cat_cols', required=False, type=int, default=0,
+                   help='count of cathegorical collumns in the dataset')
+parser.add_argument('--n-time-cols', dest='n_time_cols', required=False, type=int, default=0,
+                   help='count of time collumns in the dataset')
+
+
+args = parser.parse_args()
+
+known_cols = args.n_num_cols + args.n_string_cols + args.n_uuid_cols + args.n_cat_cols + args.n_time_cols
+
+if (args.ncols < known_cols):
+    print("count of all columns can not be lower then sum of counts of different types of columns")
+    print()
```

Comment:
Knit: `\n` at the end of the previous line ?

Reply:
It's there now